shopping-cart-client/src/app/signin/signin.component.ts

```diff
@@ -0,0 +1,36 @@
+import { Component, OnInit } from '@angular/core';
+import { NgForm } from '@angular/forms';
+import { Router } from '@angular/router';
+import { LoginService } from '../services/login-services/login.service';
+
+@Component({
+  selector: 'app-signin',
+  templateUrl: './signin.component.html',
+  styleUrls: ['./signin.component.css']
+})
+export class SigninComponent implements OnInit {
+ user={
```

Comment:
is this `user` object getting used in the template?

Reply:
No, but it is used here to keep the email and password for further use.